client/plots/sampleView.js

```diff
@@ -273,40 +274,51 @@ class SampleView {
 		if (this.settings.showDictionary) this.renderSampleDictionary()
 	}
 
-	async setControls() {
+	async setControls(q) {
 		this.dom.controlsDiv.selectAll('*').remove()
-
+		const showBrainMRI = JSON.parse(sessionStorage.getItem('optionalFeatures') || `{}`)?.showBrainMRI
+		const inputs = [
+			{
+				boxLabel: 'Visible',
+				label: 'Dictionary',
+				type: 'checkbox',
+				chartType: 'sampleView',
+				settingsKey: 'showDictionary',
+				title: `Option to show/hide dictionary table with sample values`
+			},
+			{
+				boxLabel: 'Visible',
+				label: 'Disco plot',
+				type: 'checkbox',
+				chartType: 'sampleView',
+				settingsKey: 'showDisco',
+				title: `Option to show/hide disco plots`
+			},
+			{
+				boxLabel: 'Visible',
+				label: 'Single sample',
+				type: 'checkbox',
+				chartType: 'sampleView',
+				settingsKey: 'showSingleSample',
+				title: `Option to show/hide single sample plots`
+			}
+		]
+		if (q.NIdata && showBrainMRI) {
+			inputs.push({
+				boxLabel: 'Visible',
+				label: 'brain MRI',
```

Comment:
rather than hardcoding values, define such labels in ds.queries.NIdata{} and pass to client via termdbConfig